src/puppetlabs/puppetdb/command/dlo.clj

```diff
@@ -71,31 +76,43 @@
                           (.printStackTrace exception out))
                         attempts))))
 
+(defn- strip-metadata-suffix
+  [queue-metadata-str]
+  (str/replace queue-metadata-str #"\.json$" ""))
+
 (defn- entry-cmd-data-filename
   [entry]
-  ;; 10291-1469469689-cat-4-foo.org.json
+  ;; 10291_1469469689_cat_4_foo.org.json
```

Comment:
I'm a little ambivalent - I suppose it does make a clear (and pointless?) distinction between which bits are stockpile and which pdb, but yeah, assuming Ryan didn't pick _ to intentionally differ, it's mostly a "historical" accident; I originally just chose dash as a personal preference.